packages/wasm-utils/src/utxo.rs

```diff
@@ -93,7 +93,7 @@ impl JsUtxo {
 
 	pub fn get_chain_id_bytes(&self) -> Vec<u8> {
 		match &self.inner {
-			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.chain_id.into_repr().to_bytes_le(),
+			JsUtxoInner::Bn254(bn254_utxo) => bn254_utxo.chain_id.into_repr().to_bytes_be(),
```

Comment:
We should use the method on the `u64` type instead, since we also have an implementation for getting the raw value.

```
pub fn get_chain_id_bytes(&self) -> Vec<u8> {
        self.get_chain_id_raw().to_be_bytes()
}
```

Reply:
This didn't work well with the proof generation